composer.json

```diff
@@ -41,6 +41,7 @@
             "cache incr",
             "cache replace",
             "cache set",
+            "cache supports",
```

Comment:
@danielbachhuber I'm guessing that you add this line, run some command and it updates the README.
Can you show me how to do this?
Might come in handy next time.

Reply:
After adding that line `README.md` regeneration can be even skipped and Github Actions will create a separate  PR after the main PR is merged - see https://github.com/wp-cli/.github/blob/main/.github/workflows/reusable-regenerate-readme.yml

Reply:
> After adding that line `README.md` regeneration can be even skipped and Github Actions will create a separate PR after the main PR is merged - see https://github.com/wp-cli/.github/blob/main/.github/workflows/reusable-regenerate-readme.yml

Correct. However, to avoid the separate PR (which was unnecessary because I was already in the code), I simply ran `wp scaffold package-readme ./` in the repository directory.